po/de.po

```diff
@@ -518,41 +507,41 @@ msgstr "Sperren der Geschichts-Datei dauerte zu lang (%.3f Sekunden)."
 #: src/history.cpp:887
 #, c-format
 msgid "Error when renaming history file: %s"
-msgstr ""
+msgstr "Fehler beim Umbenennen der History-Datei: %s"
 
 #: src/history.cpp:1285
 #, c-format
 msgid "History session ID '%ls' is not a valid variable name. "
-msgstr ""
+msgstr "History Sitzungs-ID '%ls' ist kein gültiger Variablenname."
 
 #: src/io.cpp:25
 #, c-format
 msgid "An error occurred while redirecting file '%ls'"
-msgstr ""
+msgstr "Bei der Umleitung der Datei '%ls' ist ein Fehler aufgetreten"
 
 #: src/io.cpp:26
 #, c-format
 msgid "The file '%ls' already exists"
-msgstr ""
+msgstr "Die Datei '%ls' existiert bereits"
 
 #: src/io.cpp:254
 #, c-format
 msgid "Path '%ls' is not a directory"
-msgstr ""
+msgstr "Pfad '%ls' ist kein Ordner"
 
 #: src/io.cpp:257
 #, c-format
 msgid "Path '%ls' does not exist"
-msgstr ""
+msgstr "Pfad '%ls' nicht gefunden"
 
 #: src/output.cpp:434
 #, c-format
 msgid "Tried to use terminfo string %s on line %ld of %s, which is "
-msgstr ""
+msgstr "Versucht, terminfo Text %s in Zeile %ld von %s zu nutzen, dieser ist"
```

Comment:
Not entirely sure how gettext works here, but this is missing a translation for the second half of the sentence.

Reply:
Ah, this has failed to extract the entire string because of a linebreak:

https://github.com/fish-shell/fish-shell/blob/4ed74ed6c16487f4f18e5933c8d4f2e51d83800e/src/output.cpp#L436-L437

Reply:
Ok i've translated the whole message now